weaver/wps_restapi/swagger_definitions.py

```diff
@@ -354,6 +362,13 @@ class URL(ExtendedSchemaNode):
     format = "url"
 
 
+class NRST(ExtendedSchemaNode):
+    schema_type = String
+    description = "Linked to an output pattern."
+    example = "some-object-slug-name"
```

Comment:
I don't think this example properly represents what this schema attempts to handle.